auslib/migrate/versions/018_add_scheduled_rule_changes_signoffs.py

```diff
@@ -0,0 +1,35 @@
+from sqlalchemy import Table, Column, Integer, String, MetaData, \
+    BigInteger
+
+
+def upgrade(migrate_engine):
+    metadata = MetaData(bind=migrate_engine)
+
+    rules_scheduled_changes_signoffs = Table( # noqa - to hush pyflakes about this not being used.
+        "rules_scheduled_changes_signoffs", metadata,
+        Column("sc_id", Integer, primary_key=True, autoincrement=False),
+        Column("username", String(100), primary_key=True),
+        Column("role", String(50), nullable=False),
+    )
+
+    rules_scheduled_changes_signoffs_history = Table(
+        "rules_scheduled_changes_signoffs_history", metadata,
+        Column("change_id", Integer, primary_key=True, autoincrement=True),
+        Column("changed_by", String(100), nullable=False),
+        Column("sc_id", Integer, nullable=False, autoincrement=False),
+        Column("username", String(100), nullable=False),
+        Column("role", String(50)),
+    )
+    if migrate_engine.name == "mysql":
```

Comment:
Maybe...I'm a bit concerned about having these scripts depend on shared code because it may make it easier to break an older version. OTOH, we use this pattern *everywhere*.